jetty-websocket/websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/internal/util/JettyMethodHandle.java

```diff
@@ -0,0 +1,46 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.websocket.core.internal.util;
+
+import java.lang.invoke.MethodHandle;
+
+/**
+ * Jetty interface for managing {@link MethodHandle}s.
+ *
+ * This differs from {@link MethodHandle}s that any calls the methods such as {@link #bindTo(Object)}
+ * will change the instance of {@link JettyMethodHandle} it is called on.
+ */
+public interface JettyMethodHandle
```

Comment:
I don't like the name `JettyMethodHandle` as it is just using `Jetty` as a meaningless prefix.

Something like `MethodHolder` would be better.  We have lots of `Holder` classes in jetty, so the concept is well known... it's a class that holds a reference to another class and provides some services for that.  Specially as it appears that some impls don't even use `MethodHandle`, so that should not be in the name.

Reply:
Renamed to `MethodHolder`.